lib/watir/elements/select.rb

```diff
@@ -179,6 +190,19 @@ def process_str_or_rx(str_or_rx)
       end
     end
 
+    def type_check(value)
+      msg = "expected String, Number or Regexp, got #{value.inspect}:#{value.class}"
```

Comment:
Do we actually support Numeric? As much as it's in the old `#find_options` case statement, I don't think we actually support it.

Reply:
No, we don't. I removed it from the code and forgot to remove it from the message.

Reply:
Actually, I went ahead and added support for Numeric because it was pretty trivial, and why not.